numpy/random/mtrand/Python.pxi

```diff
@@ -15,7 +15,7 @@ cdef extern from "Python.h":
     object PyString_FromStringAndSize(char* c_string, int length)
 
     # Float API
-    double PyFloat_AsDouble(object ob)
+    double PyFloat_AsDouble(object ob) except? -1.0
```

Comment:
This causes cython to always check for raised exceptions when `PyFloat_AsDouble` returns -1.0.

This is what is recommended in the python API docs:
https://docs.python.org/2/c-api/float.html#c.PyFloat_AsDouble